conf/modules/logger_sd_spi_direct.xml

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+
+<module name="logger_sd_spi_direct" dir="loggers">
+  <doc>
+    <description>Direct SPI SD Logger that saves pprzlog messages to SD Card.</description>
+    <configure name="SDLOGGER_DIRECT_SPI" value="SPI1|SPI2|SPI3|SPI4|SPI5|SPI6" description="Port to which the SD Card is connected."/>
+    <configure name="SDLOGGER_DIRECT_SPI_SLAVE" value="SPI_SLAVE1|SPI_SLAVE2|SPI_SLAVE3|SPI_SLAVE4|SPI_SLAVE5|SPI_SLAVE6" description="Port to which the SD Card is connected."/>
+    <configure name="LOGGER_CONTROL_SWITCH" value="RADIO_AUX2"/>
+    <configure name="LOGGER_LED" value="none"/>
+  </doc>
+  <settings>
+    <dl_settings NAME="sdlogger">
+      <dl_settings NAME="SD Logger">
+        <dl_setting module="loggers/sdlogger_spi_direct" var="sdlogger_spi.status" min="0" max="20" step="1"/>
+        <dl_setting module="loggers/sdlogger_spi_direct" var="sdcard1.status" min="0" max="40" step="1"/>
+        <dl_setting module="loggers/sdlogger_spi_direct" var="sdcard1.error_status" min="0" max="15" step="1"/>
+        <dl_setting module="loggers/sdlogger_spi_direct" var="sdlogger_spi.command" min="1" max="42" step="1" />
+      </dl_settings>
+    </dl_settings>
+  </settings>
+  <header>
+    <file name="sdlogger_spi_direct.h"/>
+  </header>
+  <init fun="sdlogger_spi_direct_init()"/>
+  <periodic fun="sdlogger_spi_direct_periodic()" freq="512" start="sdlogger_spi_direct_start()" stop="sdlogger_spi_direct_stop()" autorun="TRUE"/>
+  <datalink message="SETTING" fun="sdlogger_spi_direct_command()"/>
+  <makefile>
```

Comment:
Would make sense to set the target to `ap` here and merge the other makefile section into it, otherwise it will try to build the module for nps, sim, etc as well and fail....